src/prometheus/deploy/prometheus-configmap.yaml.template

```diff
@@ -32,7 +32,7 @@ data:
     - job_name: 'pai_serivce_exporter'
       scrape_interval: {{ prom_info["scrape_interval"] }}s
       kubernetes_sd_configs:
-      - api_server: "{{ cluster_cfg["kubernetes"]["api-servers-url"] }}"
+      - api_server: {{ cluster_cfg["cluster"]["kubernetes"]["api-servers-url"] }}
```

Comment:
I think kubernetes should have a independent key, rather than a child key of cluster.

Reply:
Hi @ydye,
I haven't decided where to place it.
Now, put it under 'cluster' because don't need to create a separately config parser.
Will discuss it later.